packages/eslint-plugin/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "src"
+  },
+  "include": ["src", "src/rules"],
+  "references": [{ "path": "../header-parser" }]
```

Comment:
shouldn't packages/dtslint/tsconfig.json need to add a dependency on "../eslint-plugin"?